src/NavBar.ts

```diff
@@ -64,35 +69,6 @@ export default class NavBar {
 
             return point
         })
-
-        // support legacy URLs without coordinates (not initialized) and only text, see #199
-        if (points.some(p => !p.isInitialized && p.queryText.length > 0)) {
-            if (!profile.name) profile = { name: 'car' }
-            let fullyInitPoints: QueryPoint[] = Array.from({ length: points.length })
-            points.forEach((p, idx) => {
-                if (p.isInitialized) fullyInitPoints[idx] = p
-                else
-                    getApi()
-                        .geocode(p.queryText, 'nominatim')
```

Comment:
Here we got console errors in cases like `point=`, because apparently we get an error when geocoding with an empty string.